hal/src/can.rs

```diff
@@ -149,18 +150,22 @@ impl TxPin for Pin<PB2, PeripheralA> {
     type ValidFor = Can0;
 }
 
+#[cfg(feature = "pins-144")]
 impl RxPin for Pin<PC12, PeripheralC> {
     type ValidFor = Can1;
 }
 
+#[cfg(any(feature = "pins-100", feature = "pins-144"))]
 impl RxPin for Pin<PD28, PeripheralB> {
```

Comment:
PD28 is only listed as CANRX1 on the 100 pin version.

Reply:
Smells like an error in the datasheet to me, but if that's the case it should be verified that it works on the 144 pin version as well.

Reply:
The API does not model this one-to-many input relationship, no.

Reply:
Such a poor HW design decision. Shame. Thanks for tuning in @tmplt